From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@LuisCabrer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (luisca). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Lisaco88 committed:
edit pass: virtual-network-network-interface-vm

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into virtual-network-network-interface-vm

Lisaco88 committed:
edit pass: virtual-network-network-interface-vm

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into virtual-network-network-interface-vm

Lisaco88 committed:
edit pass: virtual-network-network-interface-vm

Commit:
initial authoring

carolinacmoravia committed:
M22746: Should text "This link has broken (...)" get removed?

Hello, @YuriDio,

Translator has reported possible source content issue.  We assume that the text in brackets should get changed before release: (This link has broken. Request you to update with the correct URL).

Please, help to check my comment tag added into the article and reply with explanation if source text should be fixed, or it is by design. If fix is needed, please, let me know either if you would like me to fix it in this PR, or if you will fix it in another PR. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

asmalser-msft committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master'

v-dotren committed:
Task 1164455: Wrote new samples page for Java

jomolesk committed:
Content name update

jomolesk committed:
Minor content update

v-dotren committed:
Incorporated dev review feedback

Commit:
initial draft

Commit:
adding link to temporary repo for project

jomolesk committed:
Additional content update

Commit:
fixing links

Commit:
acrolinx

Commit:
fixing steps

Commit:
adding images, doing some revision

Commit:
Manage route tables.

Commit:
Fixes.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into manage-route-tables

Commit:
Fixes.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into manage-route-tables

jomolesk committed:
Product name spelling update

asmalser-msft committed:
Updating what happens during provisioning

asmalser-msft committed:
Updating what happens during provisioning

asmalser-msft committed:
Update

asmalser-msft committed:
Typo

asmalser-msft committed:
Update

asmalser-msft committed:
typo

asmalser-msft committed:
Update

asmalser-msft committed:
typo

asmalser-msft committed:
grammar

asmalser-msft committed:
Update

asmalser-msft committed:
Added link to API docs

asmalser-msft committed:
Update

asmalser-msft committed:
Update

asmalser-msft committed:
FIxed typo in link

asmalser-msft committed:
Update

Commit:
Fixes.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into manage-route-tables

Commit:
Update perms.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into manage-route-tables

Commit:
changing h1

Commit:
Refreshed and added service endpoints

Commit:
Updates

v-geberr committed:
regions page

Commit:
updates to better/private IP range for HDInsight.

Commit:
linking

v-shils committed:
Merge pull request #33118 from jimdial/patch-72

Refreshed and added service endpoints

PRMerger19 committed:
Merge pull request #33122 from v-geberr/0209-regions-2

regions page

Commit:
fixing URI

Commit:
Refresh and updates.

Commit:
Fixes.

Commit:
dhanyah01

v-geberr committed:
finished with region replacement

MGoedtel committed:
Updated quickstart article to streamline steps

Commit:
updates for service fabric 6.1CU1 release

PRMerger16 committed:
Merge pull request #33124 from MarkusVi/dhanyah01

dhanyah01

v-shils committed:
Merge pull request #33054 from Blackmist/kafkaaks

Kafka on HDInsight to AKS

MGoedtel committed:
minor revision to article

jomolesk committed:
De-localization edits

PRMerger18 committed:
Merge pull request #33126 from v-payc/changesFor6.1CU1

updates for service fabric 6.1CU1 release

Commit:
Refresh and removed cross-sub preview info.

Commit:
Fix

Commit:
dhanyah02

Commit:
Removed registration and preview text.

Commit:
Fix

MGoedtel committed:
updated TOC

Commit:
Fixes.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into manage-route-tables

v-shils committed:
Merge pull request #33121 from jimdial/patch-73

Updates

asmalser-msft committed:
Applying feedback

MGoedtel committed:
Corrected formatting issue

v-geberr committed:
fix md

Commit:
dhanyah02

v-geberr committed:
acro

v-shils committed:
Merge pull request #33046 from jomolesk/master

Content name update

Commit:
Merge pull request #33125 from v-geberr/0209-regions-3

finished with region replacement

v-shils committed:
Merge pull request #33127 from MGoedtel/UpdateLinuxQuick29

Updated quickstart article to streamline steps

v-dotren committed:
Updated TOC to add sql-api-java-samples.md

mrbullwinkle committed:
minor fixes

mrbullwinkle committed:
add ms.author

YuriDio committed:
small update

asmalser-msft committed:
Removing references to en-us locale in URLs

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into virtual-network-network-interface-vm

asmalser-msft committed:
Typo

Commit:
Merge pull request #32852 from carolinacmoravia/patch-601

M22746: Should text "This link has broken (...)" get removed?

Lisaco88 committed:
edit pass: virtual-network-network-interface-vm

rjagiewich committed:
Merge pull request #33130 from MarkusVi/dhanyah02

dhanyah02

GitHubber17 committed:
Merge pull request #33011 from v-dotren/dt-cosmosdb-java-samples

Cosmos DB new Java samples page

rjagiewich committed:
Merge pull request #33128 from jimdial/patch-74

Refresh and removed cross-sub preview info.

GitHubber17 committed:
Merge pull request #33058 from jimdial/manage-route-tables

Manage route tables.

GitHubber17 committed:
Merge pull request #33129 from jimdial/patch-75

Removed registration and preview text.

rjagiewich committed:
Merge pull request #33134 from YuriDio/Feb2018Updatess

small update

rjagiewich committed:
Merge pull request #33095 from asmalser-msft/master

Updating what happens during provisioning

ShawnJackson committed:
Merge pull request #32708 from Lisaco88/virtual-network-network-interface-vm

edit pass: virtual-network-network-interface-vm

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

v-alje committed:
Merge pull request #33139 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger20 committed:
Merge pull request #33133 from mrbullwinkle/mrb_02_09_2018_web_app_availability

minor fixes
